Sources/StreamChatUI/Common Views/CurrentChatUserAvatarView_Tests.swift

```diff
@@ -10,27 +10,39 @@ import XCTest
 class CurrentChatUserAvatarView_Tests: XCTestCase {
     var defaultAvatarView: CurrentChatUserAvatarView! = CurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
     var customAvatarView: CustomCurrentChatUserAvatarView! = CustomCurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
+    var defaultAppearanceHookAvatarView: CurrentChatUserAvatarView! = CurrentChatUserAvatarView().withoutAutoresizingMaskConstraints
```

Comment:
You can't do this in tests. These values must be set and reset in `setUp` and `tearDown` methods. Please look at LLC tests how it's done there.

Reply:
Hmm. Why even keep those locally in test? I changed this a bit so the values are created just in scope of the test. Let's see...